otter/utils.py

```diff
@@ -364,6 +380,31 @@ def _get_logger(cls):
         return cls._logger_instance
 
 
+class QueueLoggingHandler(logging.Handler):
+    """The logging handler that writes INFO messages to the log_queue.
+    This handler is set up when the result_queue is passed to
+    `otter grade`
```

Comment:
Change the top of the docstring to

```
"""
This logging handler writes log messages to a ``multiprocessing.Queue``.
```

The information about it being set up in otter grade is unnecessary since this could be used by other components.